slides/kube/kubectlrun.md

```diff
@@ -232,15 +233,16 @@ Unfortunately, `--follow` cannot (yet) be used to stream the logs from multiple
 
 ---
 
-class: title
-
-Meanwhile,
-<br/>
-at the Google NOC ...
-<br/>
-<br/>
-.small[“Why the hell]
-<br/>
-.small[are we getting 1000 packets per second]
-<br/>
-.small[of ICMP ECHO traffic from these IPs?!?”]
+## Aren't we flooding 1.1.1.1?
+
+- If you're asking yourself that question:
+  congratulations for being a good citizen!
```

Comment:
Hmm. I find "citizen" (unmodified) to be a bit questionable (especially giving a training in the US, with all its political terribleness). Maybe "good Internet citizen"? Or "thank you for having oncall empathy" or something?

Reply:
good Internet citizen sounds good! We can also rephrase to sidestep the issue entirely... e.g. "Good question!" or something like that?

Reply:
I like that!